firestore/integration_test_internal/src/firestore_integration_test.h

```diff
@@ -280,37 +285,27 @@ class FirestoreIntegrationTest : public testing::Test {
 
   // TODO(zxu): add a helper function to block on signal.
 
-  // A helper function to block until the future completes.
+  // Blocks until the future completes and returns the future result.
```

Comment:
ultranit: Consider adding a comma after "completes".

Reply:
Hmm, my understanding is that this is a compound predicate (`Blocks... and returns`) and thus doesn't require a comma. I can't dig up a very authoritative reference quickly but the few links that come up in a search seem to agree -- see e.g. [this](https://grammar.yourdictionary.com/sentences/what/what-is-a-predicate.html).

Reply:
Ah, I see what you mean, and I fully agree. How about `Blocks until the future is completed and returns...`? I think it could still be read the wrong way but it seems like an improvement. What do you think?